### ISSUE TITLE
Upcoming ORT 1.20 Release Overview

### ISSUE THREAD

**Issue by sophies927** [microsoft/onnxruntime#22274]
We are planning to release ORT 1.20 at the end of October.

For more information (announcements, major features, etc.), check out the new ONNX Runtime Release Roadmap page on our website: https://onnxruntime.ai/roadmap!

Feel free to respond to this issue with any questions.

**Comment by FricoRico** [comment]
Is there already documentation I can read about the following points:

- Add Android QNN support, including a pre-build package, performance improvements, and Phi-3 model support.
- Add GPU EP support for ORT Mobile.

**Comment by sophies927** [comment]
> Is there already documentation I can read about implementing the following points:
> 
> * Add Android QNN support, including a pre-build package, performance improvements, and Phi-3 model support.
> * Add GPU EP support for ORT Mobile.

@edgchen1 @skottmckay

**Comment by edgchen1** [comment]
> Is there already documentation I can read about implementing the following points:
> 
> * Add Android QNN support, including a pre-build package, performance improvements, and Phi-3 model support.
> * Add GPU EP support for ORT Mobile.

There is existing documentation about the QNN EP. It's not Android-specific.
https://onnxruntime.ai/docs/execution-providers/QNN-ExecutionProvider.html

**Comment by einarwar** [comment]
Is there any plan for when you will be supporting python 3.13?

**Comment by Djdefrag** [comment]
Please, is this problem #20713 expected to be resolved? 

We have been waiting for 6 months....

**Comment by tianlinzx** [comment]
Please, is this problem https://github.com/microsoft/onnxruntime/issues/18942  expected to be resolved?

We have been waiting for 1year ....

**Comment by sophies927** [comment]
> Is there any plan for when you will be supporting python 3.13?

Yes, Python 3.13 support will be included in the upcoming ORT 1.20 release.

**Comment by FricoRico** [comment]
> > Is there already documentation I can read about implementing the following points:
> > 
> > * Add Android QNN support, including a pre-build package, performance improvements, and Phi-3 model support.
> > * Add GPU EP support for ORT Mobile.
> 
> There is existing documentation about the QNN EP. It's not Android-specific. https://onnxruntime.ai/docs/execution-providers/QNN-ExecutionProvider.html

What about the GPU EP for mobile?

**Comment by snnn** [comment]
> Is there any plan for when you will be supporting python 3.13?

Yes.  Our release RC packages are not ready yet, but now you can 3.13 packages from our nightly feed by using the following command:
```
python3 -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ --pre onnxruntime
```

**Comment by pdufour** [comment]
@sophies927 How  can we try wasm64 listed here?
> wasm64 support (available in custom builds but not included in released packages).

**Comment by sophies927** [comment]
> [@sophies927](https://github.com/sophies927) How can we try wasm64 listed here?
> 
> > wasm64 support (available in custom builds but not included in released packages).

@EmmaNingMS do you know?

**Comment by snnn** [comment]
The WASM64 support was in PR #21836, which  was not included in this release.

**Comment by amethys7dev** [comment]
Hoping https://github.com/microsoft/onnxruntime/issues/18942 gets resolved soon too :/

**Comment by deischi** [comment]
https://onnxruntime.ai/roadmap is getting a bit outdated. It still has 1.20.1 as "in-progress".
Is there already a more detailed timeline and content for the 1.21 release?
